sci-libs/atlas/atlas-3.10.3.recipe

```diff
@@ -0,0 +1,145 @@
+SUMMARY="Automatically Tuned Linear Algebra Software"
+DESCRIPTION="A software library for linear algebra. It provides a mature open \
+source implementation of BLAS APIs for C and Fortran77."
+HOMEPAGE="http://math-atlas.sourceforge.net/"
+COPYRIGHT="2011-2016 The ATLAS Project"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://downloads.sf.net/math-atlas/atlas$portVersion.tar.bz2"
+CHECKSUM_SHA256="2688eb733a6c5f78a18ef32144039adcd62fabce66f2eb51dd59dde806a6d2b7"
+SOURCE_DIR="ATLAS"
+PATCHES="atlas-$portVersion.patchset"
+ADDITIONAL_FILES="makefile.shared.mt
+	makefile.shared.st"
```

Comment:
Instead of adding copies of these files in the HaikuPorts repo, you could use this:
```
archlinuxCommit="286cdb8eafb3756cdb3d63f0f6c6084e55a31169"
archlinuxRepoBaseUri="https://aur.archlinux.org/cgit/aur.git/plain/"
SOURCE_FILENAME_2="makefile.shared.mt"
SOURCE_FILENAME_3="makefile.shared.st"
SOURCE_URI_2="$archlinuxRepoBaseUri/$SOURCE_FILENAME_2?h=atlas-lapack&id=$archlinuxCommit"
SOURCE_URI_3="$archlinuxRepoBaseUri/$SOURCE_FILENAME_3?h=atlas-lapack&id=$archlinuxCommit"
CHECKSUM_SHA256_2="7627446a0d598b55bc8f2795f8ff2028f134928b9ca597b76bf8dd5622dbff69"
CHECKSUM_SHA256_3="18cfd59a104451c03c3b91020786260cf2b2581642a9c8f8598ba1d26b9cb3bf"
```
* makefile.shared.mt would be available as $sourceDir2/$SOURCE_FILENAME_2
* makefile.shared.st would be available as $sourceDir3/$SOURCE_FILENAME_3